SpatialGDK/Documentation/known-issues.md

```diff
@@ -14,11 +14,8 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | Dynamic component replication. | 2018-07-03 | [UNR-366](https://improbableio.atlassian.net/browse/UNR-366) | Mimic behavior with static components. |
 | Components with replicated sub-objects fail. | 2018-07-03 |[UNR-417](https://improbableio.atlassian.net/browse/UNR-417) | Make all components top level. |
 | You can't have multiple replicated components of the same type on the same Actor. | 2018-07-03 | [UNR-416](https://improbableio.atlassian.net/browse/UNR-416) | None |
-| The [Interop Code Generator]({{urlRoot}}/content/interop.md) is not optimized. | 2018-07-13 | | None | 
 | User code must be IWYU-compliant, meaning you must import all the headers you use.| 2018-07-13 |[UNR-380](https://improbableio.atlassian.net/browse/UNR-380) | None |
-| Compilation error for RPCs that take structs with private members. | 2018-07-16 | [UNR-144](https://improbableio.atlassian.net/browse/UNR-144) | Add the type binding as a friend class. |
 |Interop codegen fails when the destination file is locked. The workflow is less than ideal as you can't run it again until you restart the Unreal Editor. | 2018-07-16 | [UNR-350](https://improbableio.atlassian.net/browse/UNR-350) | Ensure destination files/folders are unlocked. |
 | Deleting entities directly from the Inspector causes undefined behavior. | 2018-07-17 | [UNR-425](https://improbableio.atlassian.net/projects/UNR/issues/UNR-425) | None |
-| Having generated files in the symlinked folder `Source/SpatialGDK` means you can’t work in two projects at the same time using the same clone of the SpatialOS GDK for Unreal. | 2018-07-23 | [UNR-458](https://improbableio.atlassian.net/browse/UNR-458) | Clone the GDK for Unreal separately for each project. |
 | We don't support listen servers. | 2018-07-30 | | Use dedicated servers instead. |
-| Stably-named replicated actors cannot be referred to by their path after transitioning between servers | 2018-08-10 | [UNR-473](https://improbableio.atlassian.net/projects/UNR/issues/UNR-473) | None |
+| Stably-named replicated actors cannot be referred to by their path | 2018-08-10 | [UNR-473](https://improbableio.atlassian.net/projects/UNR/issues/UNR-473) | None |
```

Comment:
what a regression lol

Reply:
Wait they can't at all? What am I missing here?

Reply:
We delete all stably named actors on startup, and create them again when they come down the wire. We have a jira ticket to retro fit the stably-named functionality into the spawned actors.